cmakeModules/FindPYTHON.cmake

```diff
@@ -165,10 +165,13 @@ if(PYTHONINTERP_FOUND)
 	foreach(_PYTHON_INCLUDE_DIR ${PYTHON_INCLUDE_DIRS})
 		# get library version from patchlevel.h
 		parse_version_from_file("${_PYTHON_INCLUDE_DIR}/patchlevel.h" "#define[ \t]+PY_VERSION[ \t]+" PYTHONLIBS_VERSION)
+		# cut release candidate number, i.e. 4th version number, from PYTHONLIBS_VERSION
+		STRING(REGEX REPLACE "^([0-9]+)\\.([0-9]+)\\.([0-9]+)\\.([0-9]+)" "\\1.\\2.\\3" PYTHONLIBS_VERSION ${PYTHONLIBS_VERSION})
```

Comment:
Perhaps `STRING(REGEX REPLACE "^([0-9]+)\\.([0-9]+)\\.([0-9]+)\\.([0-9]*)" "\\1.\\2.\\3" PYTHONLIBS_VERSION ${PYTHONLIBS_VERSION})` would be better, because it catches also the default case and does not ignore it, which results of course in the same string.

Reply:
Agreed. Will do.